spec/normalize.js

```diff
@@ -6,7 +6,9 @@ console.log('node path changed');
 
 beforeEach(function(){
     browser.driver.manage().window().maximize();
-
+    browser.getCapabilities().then(function(cap) {
+        browser.browserName = cap.get('browserName');
```

Comment:
you can use: process.env.BROWSER_TYPE.toLowerCase() === 'phantomjs' like we do in protractor.conf but I don't say it's better (I'm not sure what's better) , this way it will be accessible anywhere you'll use process which is also global